src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -101,11 +101,12 @@ class LocalBackend extends Backend with SharedFileSystem with LazyLogging {
     }
   }
 
+  override def resume(backendCall: BackendCall, jobKey: JobKey): ExecutionResult = ???
```

Comment:
This is actually not a problem though that's certainly not at all clear.  I'll tidy up and add some docs.